app/views/layouts/application.html.erb

```diff
@@ -6,40 +6,34 @@
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
   <%= csrf_meta_tags %>
 </head>
-
-  <% if flash[:notice] %>
-    <div class="notice"><%= flash[:notice] %></div>
-  <% end %>
-  
   <body>
     <header class="navbar navbar-fixed-top navbar-inverse align-middle"> 
       <div class="container">
         <div class = "navbar-brand">
-          <%= link_to(image_tag("NCDB_logo_web.png", :class =>"rounded", :alt => "align-middle", height: 74, width: 310), "http://ncdiaperbank.org") %>
+          <%= link_to(image_tag("NCDB_logo_web.png", :class =>"rounded", :alt => "align-middle", height: 74, width: 310), "http://ncdiaperbank.org", target: :_blank) %>
         </div>
         <nav>
           <ul class="nav navbar-nav navbar-right align-center">
-            <li><%= link_to "About", 'http://ncdiaperbank.org/about-us-2/' %></li>
-            <li><%= link_to "Collect",   '#' %></li>
-            <li><%= link_to "Donate", hotspots_path %></li>
-            <li><%= link_to "Home", static_pages_home_path %></li>
-            <li><%= link_to "Support", 'http://ncdiaperbank.org/donate/' %></li>
+            <li><%= link_to "About", 'http://ncdiaperbank.org/about-us-2/', target: :_blank %></li>
+            <li><%= link_to "Donate", 'http://ncdiaperbank.org/donate/', target: :_blank %></li>
             <% if admin_signed_in? %>
-              <li>
-                <%= link_to('Logout', destroy_admin_session_path, method: :delete) %>        
-              </li>
+              <li><%= link_to "Create", '/hotspots' %></li>
+              <li><%= link_to('Logout', destroy_admin_session_path, method: :delete) %></li>
             <% else %>
               <li>
                 <%= link_to('Login', new_admin_session_path)  %>  
               </li>
             <% end %>
           </ul>
         </nav>
+        <section>
+          <% if flash[:notice] %>
+            <div class="notice"><%= flash[:notice] %></div>
+          <% end %>
+        </section>
```

Comment:
Can we put this section under the `</header>` so that the flash isn't in the header, but rather below it? From the doc you attached it was hard to find the flash notice when it's hidden in the header.